### ISSUE TITLE
'tspin -p empty.log' waits until file has content, which may never happen

### ISSUE THREAD

**Issue by AndydeCleyre** [bensadeh/tailspin#75]
Thanks for this project, it's great!

One unexpected behavior I've noticed (in 2.0.0) is:

```console
$ rm -f empty.log
$ touch empty.log
$ tspin -p empty.log  # hangs indefinitely
```

If I leave that hanging, and in a separate Zsh session run:

```console
$ print -rl -- one two three >empty.log
```

then the first session with the hanging `tspin` command prints

```
one
two
three
```

and exits.

I wouldn't expect `tspin -p` to wait for any/more input -- is that intentional? I would expect instead for it to print nothing and exit immediately, in the case of an empty file.

**Comment by bensadeh** [comment]
Hi @AndydeCleyre and thanks for your detailed description.

This is indeed a bug. `tspin -p` should exit immediately when it reaches the end of the file, like you expect.

Additionally, I think the behavior you're seeing is something you'd expect if you ran `tspin --follow --print` (only it would run forever), which is currently unsupported, but I now see that it might be nice to have anyway.

**Comment by bensadeh** [comment]
Fixed in 45d3b1e41f9f9c35775954d6d8e3379afbd48cb4. Will be available in the upcoming 2.1.0 release.

**Comment by bensadeh** [comment]
Just released `2.1.0`. Will close this ticket, thanks for reporting it.
